test/functionalTest/cases/1188_empty_attribute_value/empty_attribute_value.test

```diff
@@ -30,12 +30,17 @@ brokerStart CB
 --SHELL--
 
 #
+# FIXME P6: step 06 in REGEXPECT of this test must change once the
```

Comment:
Probably we should close #2451 soon (I think it is quite clear no modification is needed, see my comment on that issue: https://github.com/telefonicaid/fiware-orion/issues/2451#issuecomment-242446354) thus removing these 3 lines before merging this PR.